### ISSUE TITLE
Does not compile on latest nightly

### ISSUE THREAD

**Issue by RalfJung** [laumann/compiletest-rs#131]
I am getting this error:
```
error[E0308]: mismatched types                                                                                                                                                                                     
   --> /home/r/.cargo/registry/src/github.com-1ecc6299db9ec823/compiletest_rs-0.3.14/src/lib.rs:101:22                                                                                                             
    |                                                                                                                                                                                                              
101 |         run_ignored: config.run_ignored,                                                                                                                                                                     
    |                      ^^^^^^^^^^^^^^^^^^ expected enum `test::RunIgnored`, found bool                                                                                                                         
    |                                                                                                                                                                                                              
    = note: expected type `test::RunIgnored`                                                                                                                                                                       
               found type `bool`                                                                                                                                                                                   
                                                                                                                                                                                                                   
error: aborting due to previous error                                                                                                                                                                              
                                                                                                                                                                                                                   
For more information about this error, try `rustc --explain E0308`.                                                                                                                                                
```

**Comment by RalfJung** [comment]
Ah, seems to be fixed by https://github.com/laumann/compiletest-rs/commit/2ebb5216ab7391817cd02ddfcc8388a493692d70 but there is no new release yet.

**Comment by dtolnay** [comment]
Fixed in 0.3.15 -- although now stable is broken. https://github.com/laumann/compiletest-rs/issues/133

**Comment by laumann** [comment]
[0.3.16](https://crates.io/crates/compiletest_rs/0.3.16) is out.
